docs/data-ai/data/query.md

```diff
@@ -4,6 +4,278 @@ title: "Query data"
+
+{{% /expand%}}
+
+{{% expand "mongosh or another third-party tool for querying data. Click to see instructions." %}}
+
+[Download the `mongosh` shell](https://www.mongodb.com/try/download/shell) or another third-party tool that can connect to a MongoDB data source to follow along.
+See the [`mongosh` documentation](https://www.mongodb.com/docs/mongodb-shell/) for more information.
+
+{{% /expand%}}
+
+## Query data in the Viam app
+
+Once your data has synced, you can query your data from within the Viam app using {{< glossary_tooltip term_id="sql" text="SQL" >}} or {{< glossary_tooltip term_id="mql" text="MQL" >}}.
+
+You must have the [owner role](/cloud/rbac/) in order to query data in the Viam app.
+
+{{< table >}}
+{{% tablestep %}}
+**1. Query with SQL or MQL**
+
+Navigate to the [**Query** page](https://app.viam.com/data/query).
+Then, select either **SQL** or **MQL** from the **Query mode** dropdown menu on the right-hand side.
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**2. Run your query**
+
+This example query returns 5 readings from a component called `my-sensor`:
+
+{{< tabs >}}
+{{% tab name="SQL" %}}
+
+```sql
+SELECT * FROM readings
+WHERE component_name = 'my-sensor' LIMIT 5
+```
+
+{{% /tab %}}
+{{% tab name="MQL" %}}
+
+```mql
+[
+    { "$match": { "component_name": "my-sensor" } },
+    { "$limit": 5 }
+]
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+{{% /tablestep %}}
+{{% tablestep %}}
+**3. Review results**
+
+Click **Run query** when ready to perform your query and get matching results.
+Query results are displayed as a [JSON array](https://json-schema.org/understanding-json-schema/reference/array) below your query.
+
+{{% expand "See examples" %}}
+
+- The following shows a SQL query that filters by component name and specific column names, and its returned results:
+
+  ```sh {class="command-line" data-prompt="$" data-output="3-80"}
+  SELECT time_received, data, tags FROM readings
+  WHERE component_name = 'PM_sensor' LIMIT 2
+  [
+  {
+    "time_received": "2024-07-30 00:04:02.144 +0000 UTC",
+    "data": {
+      "readings": {
+        "units": "μg/m³",
+        "pm_10": 7.6,
+        "pm_2.5": 5.7
+      }
+    },
+    "tags": [
+      "air-quality"
+    ]
+  },
+  {
+    "time_received": "2024-07-30 00:37:22.192 +0000 UTC",
+    "data": {
+      "readings": {
+        "pm_2.5": 9.3,
+        "units": "μg/m³",
+        "pm_10": 11.5
+      }
+    },
+    "tags": [
+      "air-quality"
+    ]
+  }
+  ]
+  ```
+
+- The following shows a SQL query that returns a count of records matching the search criteria:
+
+  ```sh {class="command-line" data-prompt="$" data-output="3-80"}
+  SELECT count(*) FROM readings
+  WHERE component_name = 'PM_sensor'
+  [
+  {
+    "_1": 111550
+  }
+  ]
+  ```
+
+For more information on MQL syntax, see the [MQL (MongoDB Query Language)](https://www.mongodb.com/docs/manual/tutorial/query-documents/) documentation.
+
+{{% /expand%}}
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Configure data query
```

Comment:
This is weird here - the user would get confused why they are asked to do this. See the visualize.md file comments for h2 and h3s